src/client/javascripts/App.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import {
+    AppRegistry,
+    StyleSheet,
+    Text,
+    View,
+    Platform
+} from 'react-native';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        titleText: 'Yoda',
+        os: 'OS',
+      };
+
+    //For specific platform
+    if (Platform.OS === 'ios') {
+      this.state.os = 'IOS';
+    } else if (Platform.OS === 'android') {
+      this.state.os = 'Android API :' + Platform.Version;
+    }
+
+    console.log(Platform.OS);
+
+  }
+
+  render() {
+    return (
+
+            //To do about View
+            <View style={styles.container}>
+                                  <Text style={styles.welcome}>
+                      Welcome to React Native, {this.state.os} !
+                                  </Text>
+                                  <Text style={styles.instructions}>
+                      To get started, edit index.ios.js
+                                  </Text>
+                                  <Text style={styles.instructions}>
+                      Press Cmd+R to reload,                    {'\n'}
+                      Cmd+D or shake for dev menu
+                                  </Text>
+                          </View>
+
+      );
+  }
+}
+
+//style components
```

Comment:
// Set style components. 

Note that all the comments I'm suggesting are full sentences without the subjects. 
Set style components. 
Render the screen on View. 
Populate ...

This is the minimum. If this isn't enough, we can even say fuller sentences like:

// For each state, we populate the state using the Platform params. 

It's a good practice to write both code + comments as beautiful as possible. :)